Northstar.Client/mod/scripts/vscripts/ui/menu_ns_serverbrowser.nut

```diff
@@ -953,6 +985,11 @@ string function FillInServerModsLabel( int server )
 
 
 void function OnServerSelected( var button )
+{
+	thread _OnServerSelectedAsync(button);
+}
+
+void function _OnServerSelectedAsync( var button )
```

Comment:
```suggestion
void function OnServerSelected_Threaded( var button )
```
Normally convention here is to do the same function name, followed by `_Threaded` (probably because squirrel doesn't technically do Async things)

Reply:
tbh the convention is half assed even by respawn themselves

Reply:
Addressed in e3b3a9ce22f7559d0f81016a4c2cfcf8cb5adac8.